compiler/rustc_hir_analysis/src/impl_wf_check.rs

```diff
@@ -181,6 +181,15 @@ fn report_unused_parameter(tcx: TyCtxt<'_>, span: Span, kind: &str, name: Symbol
         name
     );
     err.span_label(span, format!("unconstrained {} parameter", kind));
+    err.span_suggestion(
+        span,
+        format!(
+            "Either remove the type parameter '{}' or make use of it, for example ` for S<{}>`.",
```

Comment:
"Make use of it" is a little misleading, the user might already use it in a where clause. I'm not sure what the best wording for this would be.

Reply:
maybe just removing the "make" and leave it Either remove the type parameter or use it?
I am not sure on the words as well... =(

Reply:
yea that sounds better